docs/source/settingup.rst

```diff
@@ -1,29 +1,91 @@
+The relevant requirements are listed below.
 
-Before being able to use the extension, you need to have DataLad installed, which relies on `git-annex`_, `git`_ and `Python`_.
-If you don't have DataLad installed yet, please follow the instructions from `the datalad handbook <http://handbook.datalad.org/en/latest/intro/installation.html>`_.
+   An OSF_ account
+       You need an OSF account to be able to interact with it. If you don't have an account yet, `register here <https://osf.io/register>`_ -- its free!
 
-- An account on the OSF
+   DataLad
+       If you don't have DataLad_ and its underlying tools (`git`_, `git-annex`_) installed yet, please follow the instructions from `the datalad handbook <http://handbook.datalad.org/en/latest/intro/installation.html>`_.
 
-You need an OSF account to be able to interact with it. If you don't have an account yet, `register here <https://osf.io/register>`_.
+   [optional] An account on a Git repository hosting site
+       You should consider having an account on one or more repository hosting sites such as `GitHub <https://github.com/join>`__ , `GitLab <https://gitlab.com/users/sign_up>`_, `Bitbucket <https://bitbucket.org/account/signup/>`_ or similar.
 
-- An account on a git repository hosting site
+Installation
+^^^^^^^^^^^^
 
-You should consider having an account on one or more repository hosting sites such as `GitHub <https://github.com/join>`__ , `GitLab <https://gitlab.com/users/sign_up>`_, `Bitbucket <https://bitbucket.org/account/signup/>`_ or similar"
+``datalad-osf`` is a Python package available on `pypi <https://pypi.org/project/datalad-osf/>`_ and installable via pip_.
 
-Installation
-------------
+.. code-block:: bash
+
+   # create and enter a new virtual environment (optional)
+   $ virtualenv --python=python3 ~/env/dl-osf
+   $ . ~/env/dl-osf/bin/activate
+   # install from PyPi
+   $ pip install datalad-osf.
+
+Getting started
+^^^^^^^^^^^^^^^
+
+Here's the gists of some of this extension's functionality.
+Checkout the :ref:`Tutorial` for more detailed demonstrations.
+
+First, :ref:`provide your credentials <authenticate>`:
+
+.. code-block:: bash
+
+   # provide your OSF credentials, ideally as a token:
+   $ datalad osf-credentials
+   You need to authenticate with 'https://osf.io' credentials. https://osf.io/settings/tokens provides information on how to gain access
+   token: <your token here>
+   You need to authenticate with 'https://osf.io' credentials. https://osf.io/settings/tokens provides information on how to gain access
+   token (repeat): <your token here>
+   osf_credentials(ok): [authenticated as <user> <e-mail>]
+
+Next, create a sibling on the OSF for a DataLad dataset of your choice.
+Chose between different sibling modes to adjust how much of your dataset can be published and how it will be displayed, adjust whether your project should be private or public, attach additional meta data, or configure local sibling properties.
```

Comment:
```suggestion
Choose between different sibling modes to adjust how much of your dataset can be published and how it will be displayed, adjust whether your project should be private or public, attach additional meta data, or configure local sibling properties.
```